server/routes/one-time-code-auth/VerifyOneTimeCodeController.ts

```diff
@@ -19,10 +19,24 @@ export default class VerifyOneTimeCodeController {
     try {
       token = await this.oneTimeCodeService.verifyOneTimeCode(form.code, req.sessionID, req.ip)
     } catch (error) {
-      req.flash('errors', [
-        { href: '#email', text: 'The code you used is no longer valid. Request a new one to sign in.' },
-      ])
-      return res.redirect('/oneTimeCode/request-code')
+      switch (error.data.errorCode.code) {
```

Comment:
The internet says don't use switch - use OOP. Bit overkill in this case so I'm going with switch.